library/css/components/header.css

```diff
@@ -143,13 +153,23 @@
   display: none;
 }
 .ui.header .g20-logo {
-  width: 8%; 
-  max-width: 15rem; 
+  width: 8%;
+  max-width: 15rem;
   cursor: pointer;
   margin-left: auto;
   border-right: 1px solid gray;
   margin-right: 1rem;
 }
+.g20-logo-department {
+  width: 130px;
```

Comment:
is it mobile responsive?

Reply:
Yes, this prevents squeezing of g20 logo